code/game/gamemodes/changeling/changeling.dm

```diff
@@ -18,6 +18,7 @@ GLOBAL_LIST_INIT(slot2type, list("head" = /obj/item/clothing/head/changeling, "w
 	antag_flag = ROLE_CHANGELING
 	restricted_jobs = list("AI", "Cyborg")
 	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Blueshield", "Brig Physician")
+	protected_species = list("ipc")

```

Comment:
❄️

Reply:
Can we not just check `NOCHANGELING`

Reply:
Pretty sure, no. Because that involves checking species traits of a species they haven't spawned in as yet. We have to check for their species choice. Look at the code for protected_species in game_modes

Reply:
What Ziiro said. No solid workaround.

We could still tie the whole `NOCHANGELING` thing to `NOSCAN` though to eliminate the define.

Reply:
NOSCAN could also be used for Vox/uncloneable races, which can be clings/clinged